### ISSUE TITLE
Account blocked from repo

### ISSUE THREAD

**Issue by bigwpixel** [ytdl-org/youtube-dl#26173]
## Checklist

- [x] I'm asking a question
- [x] I've looked through the README and FAQ for similar questions
- [x] I've searched the bugtracker for similar questions including closed ones

## Question

Have written out a bugreport to do with the `--print-json` flag, but when I clicked submit I found out that my account (@probablykasper) is blocked from the youtube-dl repo. I would like some clarity as to why, and for my account to be unblocked depending on the reason.

I found out by searching my Gmail that I responded in the now-deleted issue #24842 about a YouTube 403 Forbidden error with the following message:
```
Does --no-cache-dir work? If so, probably a duplicate of #24842
```

Other than that, the account has [1 closed and 2 open issues](https://github.com/ytdl-org/youtube-dl/issues?q=author%3Aprobablykasper), and [2 issue comments](https://github.com/ytdl-org/youtube-dl/issues?q=commenter%3Aprobablykasper). I couldn't find anything bad there.

I'm confident that I did not open any issue about the 403 error, and I don't believe responding to issues about the 403 error was a rule, so to me the most logical explanation is that the block was made by mistake.

**Comment by bigwpixel** [comment]
@dstftw Would love if this could be looked into, thanks

**Comment by bigwpixel** [comment]
@phihag @remitamine Sorry for the mention, but would really like this to be addressed, if possible

**Comment by dstftw** [comment]
You've been banned as part of mass ban for ignoring sticky thread on 403 error and flooding the tracker with 403 issues. At the very least #24842 was created by you.

**Comment by bigwpixel** [comment]
@dstftw Thanks for replying! If I recall correctly, I did read the sticky thread and thought the issue was separate, but it's hard to know when the issue is 6 months old and deleted. If that wasn't the case, I apologize for the mistake and will be more careful in the future.

Would love to be able to help by responding to my open issues, file new issues and potentially contribute in the future.
